bu-slideshow.php

```diff
@@ -103,6 +105,16 @@ static public function init() {
 		
 	}
 	
+	static public function register_cpt(){
+		$args = array(
```

Comment:
I think it's best to explicitly define all of the arguments for `register_post_type`, if for no other reason than to examine each one to ensure the defaults are what we want.